ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -364,7 +368,7 @@ Test Python Model Rest Inference Via UI (Triton on Kserve)    # robocop: off=too
     ...     as_string=${TRUE}
     Run Keyword And Continue On Failure    Verify Model Inference With Retries
     ...    ${PYTHON_MODEL_NAME}    ${INFERENCE_REST_INPUT_PYTHON}    ${EXPECTED_INFERENCE_REST_OUTPUT_PYTHON}
-    ...    token_auth=${FALSE}    project_title=${PRJ_TITLE}
+    ...    token_auth=${FALSE}    project_title=${PRJ_TITLE}     application_type=${FALSE}
```

Comment:
you don't need to pass the new argument because you've set it to FALSE as default